### ISSUE TITLE
Error while altering the replication factor and the number of partitions simultanously

### ISSUE THREAD

**Issue by teebee** [StephenSorriaux/ansible-kafka-admin#46]
## Behavior

We encounter a bug (or at least a reproducable race condition) if we alter the replication factor and the number of partitions of a topic simultanously. Altering
```
topic_partitions: 3
topic_replica_factor: 2
topic_state: present
```
to
```
topic_partitions: 10
topic_replica_factor: 3
topic_state: present
```
results in
```
fatal: [localhost]: FAILED! => changed=false 
  msg: Error while updating topic 'topic.test' partitions. Error key is UNKNOWN, An unexpected server error.. Request was CreatePartitionsRequest_v0(topic_partitions=[(topic='topic.test', new_partitions=(count=<built-in method count of tuple object at 0x7fbaccf0a188>, assignment=[[102, 100], [101, 102], [100, 101], [102, 100], [101, 102], [100, 101], [102, 100]]))], timeout=15000, validate_only=False).
```
However, the replication factor has been altered:
```
kafka-topics --zookeeper ????????:2181 --topic topic.test --describe
Topic:topic.test        PartitionCount:3        ReplicationFactor:3     Configs:
        Topic: topic.test       Partition: 0    Leader: 102     Replicas: 102,100,101   Isr: 102,101,100
        Topic: topic.test       Partition: 1    Leader: 101     Replicas: 102,100,101   Isr: 101,100,102
        Topic: topic.test       Partition: 2    Leader: 100     Replicas: 102,100,101   Isr: 100,102,101
```

## Specifications

  - Library version: 0.8.0
  - Result of `pip list` command:
```
$ pip3 list
Package      Version
------------ -------
ansible      2.9.1
asn1crypto   0.24.0
bcrypt       3.1.4
cffi         1.11.5
cryptography 2.4.2
idna         2.7
Jinja2       2.10
jmespath     0.9.4
kafka-python 1.4.4
kazoo        2.6.1
kazoo-sasl   2.6.1
MarkupSafe   1.1.0
paramiko     2.4.2
pip          18.1
pure-sasl    0.6.2
pyasn1       0.4.2
pycparser    2.19
pycrypto     2.6.1
PyNaCl       1.3.0
PyYAML       4.1
setuptools   40.6.2
six          1.12.0
```
  - Kafka version: 2.3.1
  - Python version: 
```
$ ansible --version
ansible 2.9.1
  config file = None
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.6/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.6.9 (default, Oct 17 2019, 11:10:22) [GCC 8.3.0]
```
  - OS: CentOS 7

**Comment by StephenSorriaux** [comment]
Hello @teebee,

Thanks for reporting this issue. You are right this is kind of a race condition. Updating replica_factor for a topic is async on Kafka side so we should find a way to ensure the reassignment is over before tried to update the number of partitions.

**Comment by StephenSorriaux** [comment]
Fixed in the 0.8.1 release
